common/src/main/webapp/sass/explore.scss

```diff
@@ -43,6 +41,10 @@ $tree-section-width: 300px;
 }
 /* stylelint-enable selector-class-pattern */
 
+:root {
+  color-scheme: dark;
+}
+
```

Comment:
Do we want this commited or did you use it locally?

Reply:
It was moved from styles.less, I wanted to put the widths in css variables but that doesn't work